scripts/push-docker-image.sh

```diff
@@ -77,7 +77,8 @@ for r in ${OPERATORS} ${RELOADERS}; do
 				  "${r}:${TAG}-amd64" \
 				  "${r}:${TAG}-arm64" \
 				  "${r}:${TAG}-arm" \
-				  "${r}:${TAG}-ppc64le"
+				  "${r}:${TAG}-ppc64le" \
+				  "${r}:${TAG}-s390x"
```

Comment:
This change was already introduced in https://github.com/prometheus-operator/prometheus-operator/pull/4351.

Reply:
@MonolithProjects I think we will want to rebase this against master once more and squash the commits.

Reply:
@PhilipGough no problem. will do it

Reply:
@PhilipGough now it should be hopefully ok